### PR TITLE
feat: update readme v4

### PR DESCRIPTION
**Pull request by juanpicado**
**Type:** feat

**Description:**

* A better readme parser (security, xss on readme,  homepage, emails) https://github.com/verdaccio/readme
* [x] update `theme-ui` https://github.com/verdaccio/ui/pull/60

```
docker run -it --rm --name verdaccio_v4 -p 4873:4873 verdaccio/verdaccio:update-readme-v4
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,7 +42,7 @@
     "mkdirp": "0.5.1",
     "mv": "2.1.1",
     "pkginfo": "0.4.1",
-    "request": "2.88.0",
+    "request": "2.87.0",
```
Comment (jinliming2) [inline package.json]: why we downgrade this package?

Comment (juanpicado) [reply package.json]: There is a lot of issues reported with that version, it makes fails a bunch of test for some reason. 

https://github.com/request/request/issues/3072

Comment (juanpicado) [reply package.json]: https://github.com/request/request/issues/3085

Comment (jinliming2) [reply package.json]: 👍

**Pull request merged**